website/docs/components/code-block/partials/code/component-api.md

```diff
@@ -0,0 +1,41 @@
+## Component API
+
+This component uses [prism.js](https://prismjs.com/) under the hood.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="value" @type="string">
+    The text/code content for the CodeBlock.
+  </C.Property>
+  <C.Property @name="language" @type="string" @values={{array "bash" "go" "hcl" "json" "log" "shell-session" "yaml"}}>
+    The coding language to use for syntax highlighting.
+  </C.Property>
+  <C.Property @name="hasCopyButton" @type="boolean" @default="false">
+    Used to control whether a copy button for copying the code/text content will be displayed.
+  </C.Property>
+  <C.Property @name="hasLineNumbers" @type="boolean" @default="true">
+    Used to control display of line numbers.
+  </C.Property>
+  <C.Property @name="hasLineWrapping" @type="boolean" @default="false">
+    Used to control line wrapping for lines of code. If true, lines of code will wrap to fit the available space. Otherwise, horizontal scrolling is enabled if lines overflow the available space.
```

Comment:
```suggestion
    Used to control line wrapping for lines of code. If `true`, lines of code will wrap to fit the available space. Otherwise, horizontal scrolling is enabled if lines overflow the available space.
```

Reply:
Addressed in [4ea291b](https://github.com/hashicorp/design-system/pull/1749/commits/4ea291bbca9846b2dd2d1bbe4e6151c6fca2e066)